src/setupWorker/setupWorker.ts

```diff
@@ -19,14 +19,15 @@ import { createFallbackStop } from './stop/createFallbackStop'
 import { devUtils } from '../utils/internal/devUtils'
 import { SetupApi } from '../SetupApi'
 import { mergeRight } from '../utils/internal/mergeRight'
+import { SetupWorkerApi as ISetupWorkerApi } from './glossary'
```

Comment:
I think we need to export the `SetupWorkerApi` interface from `index.ts` just as we do `SetupServerApi` from `node/index.ts`. The fact that it wasn't exported before is a mistake.

Reply:
In fact it was already the interface that was exported, and not the class !

![image](https://user-images.githubusercontent.com/101416921/203995981-efe8b77e-2ff7-4d0b-8a65-1abe46023df0.png)